src/CoreWCF.Http/tests/ContractBehaviorTests.cs

```diff
@@ -246,7 +246,8 @@ private void Variation_ByHandImplementsOther()
                 cf.Endpoint.Contract.ContractBehaviors.Add(theBehavior);
                 cf.Open();
                 string expected = "IContractBehavior:ClientContract.MyMultiFacetedBehaviorAttribute;";
-                Assert.Equal(expected, BehaviorInvokedVerifier.ValidateClientInvokedBehavior(cf.Endpoint));
+                string actual = BehaviorInvokedVerifier.ValidateClientInvokedBehavior(cf.Endpoint);
+                Assert.Equal(expected, actual);
```

Comment:
this assertion is failing when running the Theory, but does not when running only this test case. I did not get why until now.

Reply:
@mconnew The SSM package update to rc1 is causing this issue. You can reproduce by updating the SSM packages in main branch.

Reply:
Please see this draft PR that repro the issue https://github.com/CoreWCF/CoreWCF/pull/450

Reply:
Oh dear, I think I might have screwed something up in ServiceContract when I added some caching. I'll investigate and update you.

Reply:
Yep, I did. I added a feature to the client to cache the ContractDescription between instantiations of the type for the same contract. I forgot that behaviors are held by the description, which means it's caching the behaviors too. So when using theories, it creating multiple instances of ChannelFactory, one for each variation, and retaining the behaviors. You should be able to work around it for now by doing `GC.Collect(2)` twice before you instantiate the ChannelFactory. I will fix this before we release.

Reply:
Fixed in #450

Reply:
I've merged #450 now.

Reply:
Thanks please assign #459 to me.